writeit/settings.py

```diff
@@ -67,6 +67,10 @@
 # If you set this to False, Django will not use timezone-aware datetimes.
 USE_TZ = True
 
+# If the site is being served over https change this to 'https' so that links
+# to subdomains use the correct scheme.
+DEFAULT_URL_SCHEME = 'http'
+
```

Comment:
Oops, silly of me to miss that when grepping... BTW, I see that you already set `DEFAULT_URL_SCHEME = {% if enable_https %}'https'{% else %}'http'{% endif %}` in `local_settings.py` in Ansible, so that's why this was working for me anyway.